fdbrpc/Locality.h

```diff
@@ -157,6 +157,16 @@ struct LocalityData {
 		_data[key] = value;
 	}
 
+	int size() const { return _data.size(); }
+	std::pair<Standalone<StringRef>, Optional<Standalone<StringRef>>> getItem(int i) {
+		ASSERT(i < _data.size());
+		std::map<Standalone<StringRef>, Optional<Standalone<StringRef>>>::iterator iter = _data.begin();
+		while (i-- > 0) {
+			iter++;
+		}
+		return *iter;
+	}
```

Comment:
Maybe it's not a huge deal, but since we don't really need random access and it's awkward to provide, we could instead have a begin and end method to return (const) iterators into the data map.

Reply:
I thought about providing the iterator to LocalityData. I didn't use that solution because this PR is the only place that needs this functionality. I chose to keep the exposed functionality from the LocalityData minimal.